crates/proto/src/dnssec/handle.rs

```diff
@@ -100,6 +118,37 @@ impl<H: DnsHandle> DnssecDnsHandle<H> {
         self
     }
 
+    /// Set a custom validation cache size
+    ///
+    /// # Arguments
+    /// * `capacity` - the desired capacity of the DNSSEC validation cache.
+    pub fn validation_cache_size(mut self, capacity: Option<usize>) -> Self {
+        let Some(capacity) = capacity else {
+            return self;
+        };
+
+        self.validation_cache = ValidationCache::new(capacity);
+        self
+    }
+
+    /// Set custom validation TTL limits
+    ///
+    /// # Arguments
+    /// * `negative` - A range of permissible TTL values for negative responses.
+    /// * `positive` - A range of permissible TTL values for positive responses.
+    ///
+    /// Validation cache TTLs are based on the Rrset TTL value, but will be clamped
+    /// to one of these values, if specified.
+    pub fn validation_ttl_limits(
+        mut self,
+        negative: RangeInclusive<Duration>,
+        positive: RangeInclusive<Duration>,
+    ) -> Self {
+        self.validation_negative_ttl = Some(negative);
+        self.validation_positive_ttl = Some(positive);
+        self
+    }
```

Comment:
Would it be better to split this into two methods, one for the positive range and one for the negative range? It feels like since both args are `RangeInclusive<Duration>` it will be hard to tell at the call-site which is which.